packages/odyssey-react-mui/src/labs/DataFilters.tsx

```diff
@@ -468,6 +482,9 @@ const DataFilters = ({
                       setIsFiltersMenuOpen(false);
                     }}
                   >
+                    {filterPopoverCurrentFilter?.render &&
```

Comment:
I don't think you need to check if render exists... just call it, no? `undefined` should be an acceptable value here if for some reason, render isn't here?